release-notes/7.0/preview/api-diff/rc1/Microsoft.NETCore.App/7.0-rc1_System.ComponentModel.md

```diff
@@ -0,0 +1,25 @@
+# System.ComponentModel
+
+``` diff
+ namespace System.ComponentModel {
+     public abstract class CustomTypeDescriptor : ICustomTypeDescriptor {
```

Comment:
I don't think the changes in this file is correct. 

CC @eerhardt

Reply:
The changes to `GetConverter()`'s return type from `TypeConverter` to `TypeConverter?` are correct. This was changed in https://github.com/dotnet/runtime/pull/72629.

The other changes are incorrect. See 

https://github.com/dotnet/runtime/blob/main/src/libraries/System.ComponentModel.TypeConverter/ref/System.ComponentModel.TypeConverter.cs#L284-L307

Reply:
They may have been a hidden attribute that I manually had to remove.

Reply:
I could bring the attribute back, but in previous PRs I was told to remove it. The problem now is that the tool is unable to remove it along with the whole API it is decorating.

Reply:
Ok I'll only leave the GetConverter diff, remove everything else.